src/components/Pagination.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import { render } from '@testing-library/react';
+import { Pagination, LinkForPage } from "./Pagination";
+
+describe('Pagination', () => {
+  let root: HTMLElement;
+
+  beforeEach(() => {
+    root = document.createElement('main');
+    root.id = 'root';
+    document.body.append(root);
+  });
```

Comment:
I dunno if it matters but this repo uses a lot of fragment snapshots

Reply:
Idk I just copied the other specs setup